lib/util/monitored-trip.js

```diff
@@ -0,0 +1,22 @@
+export const WEEKDAYS = ['monday', 'tuesday', 'wednesday', 'thursday', 'friday']
+const ALL_DAYS = ['monday', 'tuesday', 'wednesday', 'thursday', 'friday', 'saturday', 'sunday']
+
+/**
+ * Extracts the day of week fields of an object to an array.
+ * Example: { monday: truthy, tuesday: falsy, wednesday: truthy ... } => ['monday', 'wednesday' ...]
+ */
+export function dayFieldsToArray (obj) {
```

Comment:
The `obj` variable name should be changed to something that matches expectations of the incoming variable a little better. I recommend renaming to `monitoredTrip`.

Reply:
Addressed in https://github.com/opentripplanner/otp-react-redux/pull/174/commits/d77e617082a85151315a22e478befc5629fd8ba7.